data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -103,24 +99,6 @@ def transform(self) -> None:
             self.MEDIAN_INCOME_FIELD
         ]
 
-        # TODO: handle null values for CBG median income, which are `-666666666`.
```

Comment:
Stripping all the AMI stuff out of `CensusACS` and moving it to `CensusACSMedianIncome` – it was already a separate file load here anyways.